src/items/class-features/attractive_force_(su).json

```diff
@@ -0,0 +1,115 @@
+{
+  "_id": "8IFEg4zF3e0X7HdK",
+  "name": "Attractive Force (Su)",
+  "type": "feat",
+  "img": "systems/sfrpg/images/cup/gameplay/graviton.webp",
+  "system": {
+    "type": "",
+    "ability": null,
+    "abilityMods": {
+      "parts": []
+    },
+    "actionTarget": "",
+    "actionType": "",
+    "activation": {
+      "type": "move",
+      "condition": "",
+      "cost": 1
+    },
+    "area": {
+      "effect": "",
+      "shape": "",
+      "units": "none",
+      "value": null
+    },
+    "attackBonus": 0,
+    "chatFlavor": "",
+    "critical": {
+      "effect": "",
+      "parts": []
+    },
+    "damage": {
+      "parts": []
+    },
+    "damageNotes": "",
+    "description": {
+      "chat": "",
+      "gmnotes": "",
+      "short": "",
+      "unidentified": "",
+      "value": "<p>&gt;@UUID[Compendium.sfrpg.class-features.G7p2YIGFTAU6uGjL]{Stellar Revelation}</p>\n<p><sub>&nbsp;</sub>@Icon[type:graviton]Graviton Revelation</p>\n<p>Your weapon exerts a pull that makes objects cling to it and can even stymie opponents with strands of force. As a move action, you can grant a weapon you wield, including your @UUID[Compendium.sfrpg.class-features.WWHUAfgpvk6uIgY0]{Solar Weapon}, the <strong>disarm</strong> weapon special property. This benefit lasts for 1 round or until you leave graviton mode.</p>\n<p>When you are attuned or fully attuned, if you score a critical hit with the weapon, your target must succeed at a Reflex save or be @UUID[Compendium.sfrpg.conditions.nFBcsp5FS1Wi46Fc]{Entangled} for 1 round. If the weapon already has a critical hit effect, when you score a critical hit, you can apply either the weapon’s normal critical hit effect or the entangling effect.</p>"
```

Comment:
Clean up the empty `<sub>` element before the Graviton icon, and add a space between the icon and the label. Same is true on other items